src/pages/notFoundPage/NotFoundPage.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './NotFoundPage.scss';
+// import notFoundImg from '../../../public/img/page-not-found.png';
+
+export const NotFoundPage: React.FC = () => {
+  return (
+    <div className="pageNotFound__wrapper">
```

Comment:
Do this changes for all classes.
```suggestion
    <div className={classes.pageNotFound__wrapper}>
```